src/app/modules/shared-components/components/input/input.component.html

```diff
@@ -3,7 +3,7 @@
     [type]="inputType"
     [placeholder]="placeholder"
     [ngClass]="{'has-button': buttonIcon}"
-    [formControlName]="name"
+    [formControlName]="name!"
```

Comment:
I told yesterday I forbid you to use exclamation mark except for setting variable to ngprime component. 
Here input is not a ng-prime component, so it could cause problem.
If it must be a string  (initially you put ` string | number | null`? ), just put an empty string as default value.

Reply:
`parentForm.get` doesn't allow number | null, so I've removed it as well to avoid issues.
I guess I could add the support for numbers but for the moment I thought this was enough to only support strings.
I've put "" as the default value.

Reply:
But at the end, it has not been changed!

Reply:
This is finally fixed now !